src/style.css

```diff
@@ -0,0 +1,21 @@
+
+.red-block {
+  background-color: red;
+}
+
+.green-block {
+  background-color: green;
+}
+
+.blue-block {
+  background-color: blue;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+section {
```

Comment:
as I wrote before "use classes instead", "move common styles to common **class**"